### PR TITLE
Updated go-gh package

### PR DESCRIPTION
**Pull request by t-dedah**
### What are you trying to accomplish?

Updated go-gh package due to addition of  support for secure token storage in go-gh which might break the old code.

Fixes this - https://github.com/actions/gh-actions-cache/issues/59

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,31 +5,34 @@ go 1.18
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/TwiN/go-color v1.1.0
-	github.com/cli/go-gh v0.1.0
+	github.com/cli/go-gh v1.2.1
```
Comment (bishal-pdMSFT) [inline go.mod:8]: This is a major version change. Do our e2e tests cover scenarios to ensure that nothing is broken due to this?

**Review by KrazyKrazy8 — APPROVED** [review]

**Comment by msfjarvis** [comment]
Is this PR still being considered? The extension is currently unusable on the latest gh CLI.

**Comment by myk002** [comment]
It says:
image

But it appears that there is an approving review. Is there a reason this PR isn't being merged automatically?

**Comment by VivaLaPanda** [comment]
Pweease can we get this merged

**Comment by t-dedah** [comment]
Hi @VivaLaPanda @myk002 @realberliner75 @msfjarvis @KrazyKrazy8 I have created a fork for the project and released a new version `1.0.5` with the above PR. If the current PR is a blocker feel free to use https://github.com/t-dedah/gh-actions-cache.

**Comment by cwille97** [comment]
👋 Hi, sorry for the delay. My team is taking a look at this now.

**Review by sa50511 — APPROVED** [review]

**Review by sa50511 — APPROVED** [review]

**Review by sa50511 — APPROVED** [review]

**Comment by tamird** [comment]
Looks like #74 already did this.

Can you release a new version of the extension? It remains unusable on the last gh cli.

**Pull request closed without merging**